tests/models/clvp/test_tokenization_clvp.py

```diff
@@ -79,7 +79,7 @@ def get_tokenizer(self, **kwargs):
     # Copied from transformers.tests.models.gpt2.test_tokenization_gpt2.GPT2TokenizationTest.get_input_output_texts
     def get_input_output_texts(self, tokenizer):
         input_text = "lower newer"
-        output_text = "lower newer"
+        output_text = "lower[SPACE]newer"
```

Comment:
testing sets a small vocab here, so this should be the expected behaviour. see unmodified test `ClvpTokenizationTest --> test_full_tokenizer` for an example where `[SPACE]` was expected